internal/peer/node/start.go

```diff
@@ -496,13 +496,9 @@ func serve(args []string) error {
 		HandlerRegistry: chaincodeHandlerRegistry,
 	}
 	var client *docker.Client
```

Comment:
we have fallback logic in https://github.com/hyperledger/fabric/blob/b430f818b3ab955776255ca8ac2f977aa8f409b1/core/container/container.go#L107-L118 and if we leave `var client *docker.Client* nil here, that would trigger a panic. I think we'll need to change the logic in `DockerVM.Build` to error out

Reply:
Done